knowledge/pop_culture/events/barbenheimer/qna.yaml

```diff
@@ -0,0 +1,24 @@
+created_by: jennbergs
+domain: movies
+seed_examples:
+- answer: | 
+  Warner Bros. Pictures' Barbie and Universal Pictures' Oppenheimer
+  question: What two movies both launched on July 21, 2023, triggering the phenomenon known as Barbenheimer?
+- answer: |
+  Both Barbie and Oppenheimer address the idea of the "Human Era" or Anthropocene Era along with the philosophical concept of existentialism.
+  question: What is a key similarity between the two very different movies?
+- answer: |
+  On the surface, there is more different about the movies than there is similar. Oppenheimer is a historical movie dealing with the creation of the first atomic bombs, while Barbie is a movie about plastic dolls that promoted an ideal of femininity that is physically impossible to attain.
+  question: How are the two movies different?
```

Comment:
This question needs a little more focus. Something like "How are the movie Oppenheimer and Barbie different." the model needs context for these questions and something this general will choose _any_ two movies.